src/compiler/transformers/module/module.ts

```diff
@@ -1847,9 +1880,20 @@ namespace ts {
 var __importStar = (this && this.__importStar) || function (mod) {
     if (mod && mod.__esModule) return mod;
     var result = {};
-    if (mod != null) for (var k in mod) if (Object.hasOwnProperty.call(mod, k)) result[k] = mod[k];
+    if (mod != null) for (var k in mod) b(k);
     result["default"] = mod;

```

Comment:
This needs to use `Object.defineProperty` (with `value` instead of `get`), because assignments will fail if `mod` has a `default` property